### PR TITLE
3.12.2-x PR

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @craigcomstock can review this?

**Comment by Lex-2008** [comment]
@cf-bottom does it build? on exotics?

**Comment by Lex-2008** [comment]
test build: https://ci.cfengine.com/job/pr-pipeline/2393/

File: deps-packaging/postgresql-hub/cfbuild-postgresql-hub.spec [hunk 1]
```diff
@@ -6,6 +6,8 @@ Version: %{version}
 Release: 1
 Source0: postgresql-%{postgresql_version}.tar.gz
 Source1: postgresql.conf.cfengine.patch
+Source2: CVE-2019-10164.1.patch
+Source3: CVE-2019-10164.2.patch
```
Comment (vpodzime) [inline deps-packaging/postgresql-hub/cfbuild-postgresql-hub.spec]: These should be `Patch0` and `Patch1` and then you can use `%patch0` and `%patch1` macros to apply them. Check some other spec files, we are surely doing that somewhere else.

Comment (Lex-2008) [reply deps-packaging/postgresql-hub/cfbuild-postgresql-hub.spec]: indeed!

**Comment by Lex-2008** [comment]
new build: https://ci.cfengine.com/job/pr-pipeline/2403/

**Comment by Lex-2008** [comment]
new build: https://ci.cfengine.com/job/pr-pipeline/2405/

**Comment by Lex-2008** [comment]
Build approved by Jenkins: https://ci.cfengine.com/job/pr-pipeline/2405/

**Pull request merged**